addOns/authhelper/src/main/java/org/zaproxy/addon/authhelper/AuthUtils.java

```diff
@@ -909,8 +909,8 @@ public static Map<String, SessionToken> getAllTokens(HttpMessage msg, boolean in
     }
 
     /**
-     * Returns all of the identified session tokens in a request. This method looks for
-     * Authorization headers and cookies with a value over a minimum length.
+     * Returns all of the identified session tokens in a request. This method looks for any headers
+     * with "auth" in their names and cookies with a value over a minimum length.
```

Comment:
Add or update a test for this to ensure we don't break it in the future?

Reply:
I know "Authorization" contains auth, but still....

Reply:
The integration test covers this (not that I'm against unit tests).

Reply:
Updated a unit test as well 😁